index.html

```diff
@@ -0,0 +1,202 @@
+          <li><a href="#">Back End Dev</a></li>
+          <li><i class="fa-sharp fa-solid fa-circle"></i></li>
+          <li><a href="#">2015</a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="all">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JAVASCRIPT</li>
+        </ul>
+        <br>
+        <br>
+        <a href="https://github.com/citec-47/PORTFOLIO-TWO/blob/set-up/README.md" class="resume">See Project</a>
+      </div>
+  </div>
+ </div>
+ </section>
+<section class="about_me">
+  <h3 class="about">About
+    Myself</h3>
+    <p class="abouts">Hello I’m a software developer! 
+      I can help you build a product , feature or
+       website Look through some of my work and experience!
+        If you like what you see and have a project
+         you need coded, don’t hestiate to contact me.</p>
+    <section class="contact_list1">
+      <h3 class="lets">LET’S CONNECT</h3>
+      <ul class="list_holder">
+          <li class="list"><a href="https://twitter.com/home"><i class="fa-brands fa-twitter"></i></a></li>
+          <li class="list"><a href="https://www.linkedin.com/feed/?trk=homepage-basic_google-one-tap-submit"><i class="fa-brands fa-linkedin-in"></i></a></li>
+          <li class="list"><a href="#"><i class="fa-solid fa-m"></i></a></li>
+          <li class="list"><a href="https://github.com/citec-47"><i class="fa-brands fa-github"></i></a></li>
+          <li class="list"><a href="https://www.facebook.com/ndonyi.maurice"><i class="fa-brands fa-facebook"></i></a></li>
+       </ul>
+      </section>
+      <button class="resume">Get my resume</button>
+ <section class="languages">
+  <h4>Languages</h4>
+  <div class="skills">
+    <div class="add_skill"><img src="Images/javascript.png"><h3>JAVASCRIPT</h3></div>
+    <div class="add_skill"><img src="Images/html.png"><h3>HTML</h3></div>
+    <div class="add_skill"><img src="Images/css.png"><h3>CSS</h3></div>
+  </div>
+  <div class="works">
+    <div class="work_skill"><h3>Frameworks</h3><img src="Images/next.png"></div>
+    <div class="work_skill"><h3>Skills</h3><img src="Images/next.png"></div>
+  </div>
+ </section>
+  </section>
+  <section class="full_form">
+    <div class="contact_me">
+      <h1>Contact me</h1>
+      <P>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</P>
+    </div>
+    <form action="post">
+      <input type="text" value="Ndonyi" placeholder="Your Name Here" id="input" required/>
+      <input type="email" value="mauricendonyi40@gmail.com" placeholder="Your Email Here" id="input" required/>
+      <textarea id="textpanel" cols="30" rows="9" placeholder="Write Your Message Here"></textarea> 
+      <submit type="submit" class="submit">Get In Touch</a></submit>
+    </form>
+  </section> 
```

Comment:
Please consider following these 2 rules in this section as it's a good practice for semantics:-

1. The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
2. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`

Following is the code representation of the above if you are confused about what the instructions mean:-


`    <ul>
      <li>
        Languages
        <ul>
          <li>JavaScript</li>
          <li>Python</li>
          <li>Java</li>
        </ul>
      </li>
      <li>Frameworks</li>
      <li>Skills</li>
    </ul>`


![image](https://user-images.githubusercontent.com/94223367/219422805-c6dd3bc7-bbc1-4dba-9c18-85bc5d823350.png)